### PR TITLE
added info about testing webhooks

### PR DESCRIPTION
**Pull request by andrecedik**

### CODE REVIEW

File: concepts/index.md [hunk 1]
```diff
@@ -195,3 +195,16 @@ specify which events should trigger sending a message to your URL.
 
 This feature is still in beta so please send a request to support@shipcloud.io
 for it to be made available for you.
+
+## Testing
+There are a lot of tools out there for testing webhooks. If you'd like a starting point check out
+this blogpost called [_60+ Tools and Services for API and Webhook Logging, Debugging, Testing,
+Monitoring, Documentation and Discovery_](http://john-sheehan.com/blog/ultimate-api-webhook-backend-service-debugging-testing-monitoring-and-discovery-tools-list).
+We'd recommend using a service like [requestb.in](http://requestb.in) or [mocky.io](http://mocky.io)
```
Comment (sfroehler) [inline concepts/index.md]: I would suggest we mention https://www.runscope.com as well. Similar to requestb.in but can do a lot more.

**Comment by sfroehler** [comment]
Looks good to me. Only suggested to add Runscope to the list of tools.
Ready to merge

File: concepts/index.md (hunk 1 above)
Comment (andrecedik) [inline concepts/index.md]: My goal was to show free tools. requestb.in is a free tool by runscope so if you want to do more than "just mock" the API you'd go over there anyways.

**Comment by sfroehler** [comment]
"Ready to merge" was already in the previous comment. ;)

**Pull request merged**